seed/java-sdk/custom-auth/src/main/java/com/seed/customAuth/errors/UnauthorizedRequestError.java

```diff
@@ -0,0 +1,18 @@
+package com.seed.customAuth.errors;
+
+import com.seed.customAuth.resources.errors.types.UnauthorizedRequestErrorBody;
+
+public class UnauthorizedRequestError extends ApiError {
+
+    private final UnauthorizedRequestErrorBody body;
+
+    public UnauthorizedRequestError(UnauthorizedRequestErrorBody body) {
+        super("UnauthorizedRequest", 401, body);
+        this.body = body;
+    }
+
+    @Override
+    public UnauthorizedRequestErrorBody body() {
```

Comment:
this makes the `body`'s return type `UnauthorizedRequestErrorBody` instead of super's `Object` type